src/services/users.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { z } from 'zod';
+import { insertUserSchema } from '../types/users';
+
+describe('service: users', function () {
+  describe('schema: insertUserSchema', function () {
+    it('should remove empty strings from user data', function () {
+      const user: z.infer<typeof insertUserSchema> = {
+        email: '',
+        username: '',
+        firstName: '',
+        lastName: '',
+        telegram: '',
+        groupIds: ['1'],
+        userAttributes: {},
+      };
+
+      const transformedUser: { [key: string]: string | null | string[] | object } =
+        insertUserSchema.parse(user);
+
+      // loop through all keys and check if they are not empty strings
+
+      for (const key of Object.keys(transformedUser)) {
+        console.log(key);
```

Comment:
Do we want to keep this log at the tests?

Reply:
yeah, might not be ideal for this patch but from https://github.com/lexicongovernance/pluraltools-backend/pull/266 and onwards, logs will only show on failed test cases allowing us to track down the error. In this test case without the log, the message looks like:

 service: users › schema: insertUserSchema › should remove empty strings from user data

expect(received).not.toBe(expected) // Object.is equality

Expected: not ""
    
not really helping to know what attribute failed